src/flashlight.c

```diff
@@ -26,25 +26,37 @@
 
 #define QCOM_ENABLE "255"
 #define QCOM_DISABLE "0"
+#define MTK_ENABLE "1"
+#define MTK_DISABLE "0"
 
 const size_t qcom_sysfs_size = 5;
 const char* const qcom_sysfs[] = {"/sys/class/leds/torch-light/brightness",
                                   "/sys/class/leds/led:flash_torch/brightness",
                                   "/sys/class/leds/flashlight/brightness",
                                   "/sys/class/leds/torch-light0/brightness",
                                   "/sys/class/leds/torch-light1/brightness"};
+const size_t mtk_sysfs_size = 1;
+const char* const mtk_sysfs[] = {"/sys/class/flashlight_core/flashlight/flashlight_torch"};
 
 const char* qcom_torch_enable = "/sys/class/leds/led:switch/brightness";
 
 char* flash_sysfs_path = NULL;
 gboolean activated = 0;
+gboolean is_qcom_torch = 0;
```

Comment:
I think enum type variable might be a better idea for the case we ever need to extend it.